src/utils/upload/fileUploadSpecifications.js

```diff
@@ -1,6 +1,6 @@
 const fileUploadSpecifications = {
   '10X Chromium': {
-    acceptedFiles: [
+    acceptedFiles: new Set([
```

Comment:
Why do you make this a set if they are hardcoded and we dont add values here?

Reply:
To check for membership efficiently

Reply:
Oh so you mean `has()` is faster than `includes()`?

Reply:
Yes, `Set#has` is `O(1)` whereas `Array#includes` is `O(n)`

Reply:
Before we created a regex from the array but this was unnecessarily complicated